### PR TITLE
Create room page

### PR DESCRIPTION
**Pull request by FabianAronsson**
Adds design and "some" logic to the room page. The idea is that this is purely design, when actually implementing authentication and SignalR, which is done in #104 and #105.

### CODE REVIEW

File: Queueomatic/Client/Pages/Room.razor [hunk 1]
```diff
@@ -0,0 +1,72 @@
+﻿@*there needs to be a check if the room owner is connecting when participantname is omitted*@
+@page "/room/{RoomId}"
```
Comment (Mojtabarezaei4) [inline Queueomatic/Client/Pages/Room.razor:2]: The path for the room should be in plural form, for instance `/rooms/23asdq`.

File: Queueomatic/Client/Pages/Room.razor [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+@page "/room/{RoomId}/{ParticipantName}"
```
Comment (Mojtabarezaei4) [inline Queueomatic/Client/Pages/Room.razor:3]: Same as above. 
Why should be a path for a participant?

File: Queueomatic/Client/Pages/Room.razor [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+                            <div class="card-body">
+                                <h5 class="text-center">@context</h5>
+                            </div>
+                        </div>
+                    </ChildContent>
+                </Dropzone>
+            </div>
+        </div>
+        <div class="col d-flex flex-column align-items-center justify-content-center mb-5">
+            <p class="header">Idling</p>
```
Comment (Mojtabarezaei4) [inline Queueomatic/Client/Pages/Room.razor:26]: The idling list should be under waiting for help in mobile view. It is more important to see them than who is idling.

File: Queueomatic/Client/Pages/Room.razor [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+                            <h5 class="text-center">@context</h5>
+                        </div>
+                    </div>
+                </ChildContent>
+            </Dropzone>
+        </div>
+    </div>
+</div>
+
+@code {
```
Comment (Mojtabarezaei4) [inline Queueomatic/Client/Pages/Room.razor:52]: The code section should be implemented as a partial class.

**Review by Mojtabarezaei4 — CHANGES_REQUESTED** [review]

File: Queueomatic/Client/Pages/Room.razor (hunk 2 above)
Comment (FabianAronsson) [reply Queueomatic/Client/Pages/Room.razor:3]: How else would you supply the page with a participant name? We decided that you enter your name on the landing page and that name needs to be used on this page.

Comment (Mojtabarezaei4) [reply Queueomatic/Client/Pages/Room.razor:3]: Yeah, you're right but we get all participants of the room from the room itself.

Comment (FabianAronsson) [reply Queueomatic/Client/Pages/Room.razor:3]: Yes we do, but that's done on the room page

Comment (Mojtabarezaei4) [reply Queueomatic/Client/Pages/Room.razor]: Why shouldn't we add the participant when we ask for thier name, already on landing page?

Comment (FabianAronsson) [reply Queueomatic/Client/Pages/Room.razor]: You could, but you would need to retrieve the created participant from the room page. How would you know which one to retrieve? 

Either you create the participant on the landing page and supply the path with a participant id or you supply the name in the route and create it in the room. 

Either way it is the same process, but I find it better to isolate as much code as possible to its respective page.

**Review by Mojtabarezaei4 — APPROVED** [review]

**Pull request merged**